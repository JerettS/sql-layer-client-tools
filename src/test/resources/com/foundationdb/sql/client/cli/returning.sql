CREATE TABLE t(s SERIAL, v VARCHAR(32));
INSERT INTO t(v) VALUES ('one'), ('two'), ('three') RETURNING *;
UPDATE t SET v=CONCAT(v,v) WHERE s=2 RETURNING v;
DELETE FROM t WHERE s=2 RETURNING v;
