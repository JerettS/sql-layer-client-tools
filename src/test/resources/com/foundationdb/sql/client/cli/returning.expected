test=> CREATE TABLE t(s SERIAL, v VARCHAR(32));
ROWS: 0
test=> INSERT INTO t(v) VALUES ('one'), ('two'), ('three') RETURNING *;
 s |   v   
---+-------
 1 | one   
 2 | two   
 3 | three 
(3 rows)

test=> UPDATE t SET v=CONCAT(v,v) WHERE s=2 RETURNING v;
   v    
--------
 twotwo 
(1 row)

test=> DELETE FROM t WHERE s=2 RETURNING v;
   v    
--------
 twotwo 
(1 row)

test=> 
